cla-backend-go/emails/v2_cla_manager_templates.go

```diff
@@ -186,16 +186,20 @@ const (
 	V2ToCLAManagerDesigneeTemplate = `
 <p>Hello {{.RecipientName}},</p>
 <p>This is a notification email from EasyCLA regarding the project(s): {{.GetProjectsOrProject}}.</p>
-<p>The following contributor is requesting to sign the CLA for the organization {{.CompanyName}}: </p>
-<p> {{.ContributorID}} ({{.ContributorName}}) </p>
-<p>Before the user contribution can be accepted, your organization must sign a CLA.
+<p>We received a request from {{.ContributorName}} ({{.ContributorEmail}}): to contribute to the above projects on behalf of your organization</p>
+<p>Before the user contribution can be accepted, your organization must sign a Corporate CLA (CCLA).The requester has stated that you would be the initial CLA Manager for this CCLA, to coordinate the signing of the CCLA and then manage the list of employees who are authorized to contribute</p>
+<p>Please complete the following steps:</p>
+<p>&emsp;1. After login, you will be redirected to the portal {{.CorporateConsole}} where you can either sign the CLA for any of the project(s): {{range $index, $projectName := .Projects}}{{if $index}},{{end}}{{$projectName.GetProjectFullURL}}{{end}}, or send it to an authorized signatory for your company. </p>
```

Comment:
For the numbered list, let's do the standard HTML numbered list:

- remove `<p></p>`
- add `<ol> and <li>` => see: https://www.w3schools.com/html/tryit.asp?filename=tryhtml_lists_ordered